src/app/notes/notes.component.css

```diff
@@ -0,0 +1,43 @@
+.note-form {
+  padding-left: 18px;
+  margin: 30px;
+  margin-top: 0px;
```

Comment:
@siddharthchavan 
Let's align this form in the center of the page. The following lines added to the class note-form should work:

`display: flex;
  flex-direction: column;
  align-items: center;`